proto/tendermint/types/params.proto

```diff
@@ -127,3 +128,13 @@ message TimeoutParams {
   // for the full commit timeout.
   bool bypass_commit_timeout = 6;
 }
+
+// ABCIParams configure functionality specific to the Application Blockchain Interface.
+message ABCIParams {
+  // vote_extensions_enable_height configures the first height during which
+  // vote extensions will be enabled. During this specified height, and for all
+  // subsequent heights, precommit messages that do not contain valid extension data
+  // will be considered invalid. Prior to this height, vote extensions will not
+  // be used or accepted by validators on the network.
```

Comment:
Could you add a sentence clarifying that `PrepareProposal` won't accept extension at that particular height?